src/projects/projects-page.html

```diff
@@ -221,8 +221,8 @@
       _tapHideProjectList: function() {
         this.showProjectsList = !this.projectSelected;
       },
-      _hideProjectList: function(pullRequestSelected) {
-        this.showProjectsList = !pullRequestSelected;
+      _showProjectList: function(subpage) {
+        this.showProjectsList = (!subpage.subpage || subpage.subpage === 'activity');
```

Comment:
When the lists are collapsed, they should stay collapsed no matter what page I click on.